src/main/index.js

```diff
@@ -549,8 +166,33 @@ ipcMain.on('set-auth-token', (event, token) => {
 });
 
 process.on('uncaughtException', error => {
-  console.error(error);
-  safeQuit();
+  dialog.showErrorBox('Error Encountered', `Caught error: ${error}`);
+  isQuitting = true;
+  if (daemon) daemon.quit();
+  app.exit(1);
+});
+
+// Force single instance application
+const isSecondInstance = app.makeSingleInstance(argv => {
+  // Protocol handler for win32
+  // argv: An array of the second instance’s (command line / deep linked) arguments
+
+  let URI;
+  if (process.platform === 'win32' && String(argv[1]).startsWith('lbry')) {
+    // Keep only command line / deep linked arguments
+    URI = argv[1].replace(/\/$/, '').replace('/#', '#');
```

Comment:
Hmm, now we're doing this same substitution in two places instead of in one function. That feels brittle (what if we find another problem with Windows URI handling and don't realize it's there in two places), and also

We might want to move this back into its own function, or at least add another comment here. Just a thought.